.github/workflows/ci_checks.yaml

```diff
@@ -99,6 +99,21 @@ jobs:
       - name: Test in firefox
         run: wasm-pack test --headless --firefox ./bindings/wallet-js
 
+  test-key-gen-lib:
+    name: Wasm-pack test key gen and encryption lib
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install wasm pack
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+
+      - name: Test in chrome
+        run: wasm-pack test --headless --chrome ./keygen
```

Comment:
Why not also test in firefox like the other job does?

Reply:
Mostly because this will only be used in electron, which is chromium. The cost is probably negligible anyways, so I can add firefox too if you think it's better.

Reply:
I see, no need then.